File: marcos006/programacao_basica/questao5.py
print(" ** Seguradora XYZ ** ")
print(" >>> SISTEMA DE AVALIAÇÃO <<< ")
qcs = int(input("Quantidade de contratos simples: "))
qcc = int(input("Quantidade de contratos completos: "))
contrato_simples = int(2300)
contrato_composto = int(3500)
lucro = (float(contrato_simples) * qcs + float(contrato_composto) * qcc) * 0.1
print("O Lucro é: R$ " + str(lucro))

File: marcos006/programacao_basica/Questao5.java
import java.util.Scanner;

public class Questao5 {
    public static void main(String[] args) {
        Scanner s;
        s = new Scanner(System.in);
        
        System.out.println("Quantidade de contratos simples: ");
        int x = s.nextInt();

        System.out.println("Quantidade de contratos completos: ");
        int y = s.nextInt();

        int w = 2300;
        int z = 3500;

        float lucro = (x * w * 0.1f) + (z * y * 0.1f);

        System.out.println("O lucro é: R$ " + lucro);
    }
}

